VimeoDeeplink/VimeoDeeplink/VimeoDeeplink.swift

```diff
@@ -0,0 +1,593 @@
+    {
+        let deeplink = VimeoDeeplink.App
+        let url = deeplink.URL
+        
+        return UIApplication.sharedApplication().canOpenURL(url)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app via a deeplink.
+     
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openVimeo() -> Bool
+    {
+        return self.openDeeplink(.App)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "all categories view" via a deeplink.
+     
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openCategories() -> Bool
+    {
+        return self.openDeeplink(.Categories)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "individual category view" for the category with `uri` via a deeplink.
+     
+     - parameter uri: The category `uri` obtained from the Vimeo API.
+
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openCategory(uri uri: String) -> Bool
+    {
+        let deeplink = VimeoDeeplink.Category(uri: uri)
+        
+        return self.openDeeplink(deeplink)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "individual channel view" for the channel with `uri` via a deeplink.
+     
+     - parameter uri: The channel `uri` obtained from the Vimeo API.
+
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openChannel(uri uri: String) -> Bool
+    {
+        let deeplink = VimeoDeeplink.Channel(uri: uri)
+        
+        return self.openDeeplink(deeplink)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "feed view" via a deeplink.
+     
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openFeed() -> Bool
+    {
+        return self.openDeeplink(.Feed)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "explore view" via a deeplink.
+     
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openExplore() -> Bool
+    {
+        return self.openDeeplink(.Explore)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "library view" via a deeplink.
+     
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openLibrary() -> Bool
+    {
+        return self.openDeeplink(.Library)
+    }
+
+    /**
+     Attempts to open the Vimeo iOS app "me view" (user profile of the logged in user) via a deeplink.
+     
+     - returns: A boolean indicating whether the Vimeo iOS app could be opened with this deeplink.
+     */
+    public static func openMe() -> Bool
+    {
+        return self.openDeeplink(.Me)
```

Comment:
super super nit: Deep link is two words. should be `openDeepLink` rather than `openDeeplink`. I've seen this in other places in the app but I've never changed it since it was kind of a large task.